contrib/depends/packages/boost.mk

```diff
@@ -1,6 +1,6 @@
 package=boost
 $(package)_version=1_73_0
-$(package)_download_path=https://gitlab.com/ArqTras/depends/-/raw/main/
+$(package)_download_path=https://boostorg.jfrog.io/artifactory/main/release/1.73.0/source/
 $(package)_file_name=boost_$($(package)_version).tar.bz2
```

Comment:
Along with both changes mention above "file_name" can be replaced with: "boost_$(subst .,_,$($(package)_version))" which will be more future_changes compatible and will require only to change package version number and sha256 hash :)